data/competencies.yml

```
@@ -368,7 +368,9 @@
 
 - id: 07715459-0423-4507-bdf1-fc397507eec7
   summary: Leads on fixing production issues
-  examples: []
+  examples: 
+    - Takes action proactively when an incident is reported on a system they support and resolves it satisfactorily
+    - Responds to critical issues raised in #ft-tech-incidents taking the innitiative to fix them and reports back
```

I think this hash will be considered a comment, we may need to quote this example:
```suggestion
    - "Responds to critical issues raised in #ft-tech-incidents taking the innitiative to fix them and reports back"
```

---

ah, wrapped it in `backticks` in the end - look ok?

---

I think we're trying to avoid using any markdown-like syntax, as it won't be rendered as markdown. I'm trying to find where we wrote this though

---

true. will update